lms/templates/student_account/login.underscore

```diff
@@ -53,6 +53,7 @@
     <% } %>
 </form>
 
+<% if ( context.createAccountOption ) %>
```

Comment:
nit: Could you please make it and if block with close `<% } %>` to improve readability?